app/schema.graphql

```diff
@@ -496,6 +496,14 @@ type Span {
   tokenCountTotal: Int
   tokenCountPrompt: Int
   tokenCountCompletion: Int
+
+  """All descendent spans in a flatten list"""
+  descendentSpansFlattenList: [Span!]!
+
+  """
+  JSON string of the adjacency list representing the family tree with the current span as the root
+  """
+  descendentTreeAdjacencyList: String!
```

Comment:
FYI I have some UI code to re-produce this in the tree view now. A nuance here that I figured out is that when building this exact tree in #1093 , I needed to actually sort the child spans by time because that was the only way to reliably get the tree to reflect execution order. Happy to take this on in the UI for now since I do have that code that I can make slightly more generic